02.md

```diff
@@ -6,9 +6,13 @@ Follow List
 
 `final` `optional`
 
-A special event with kind `3`, meaning "follow list" is defined as having a list of `p` tags, one for each of the followed/known profiles one is following.
+A special event with kind `3`, meaning "follow list", is defined as having a list of `p` tags, one for each of the profiles to be followed.
 
-Each tag entry should contain the key for the profile, a relay URL where events from that key can be found (can be set to an empty string if not needed), and a local name (or "petname") for that profile (can also be set to an empty string or not provided), i.e., `["p", <32-bytes hex key>, <main relay URL>, <petname>]`.
+Each "p" tag entry should be populated as follows: 
+- The first element is "p"
+- The second element contains the pubkey string
+- The third element is either an empty string, or a relay URL where events from that key can be found
+- The fourth and final element is either an empty string or a string with a local name for that profile (or "petname")
```

Comment:
This mostly just seems more verbose for little benefit, the notation above is very common in the NIPs repo and clear enough.

Reply:
I understand. Adjusted with far fewer words.